README.md

```diff
@@ -82,6 +85,64 @@ $ touch /var/lib/www/html/hello_world
 
 If something does not work as expected, check the troubleshooting section below.
 
+
+### Static Provision
+
+
+#### 1. Deploy the driver for static provision
+
+```bash
+cd deploy/kubernetes/static
+kubectl create -f attacher.yaml
+kubectl create -f csi-s3.yaml
+```
+
+
+#### 2. Test the S3 driver for static provision
+
+* Create a bucket in your S3 first. Then create a Persistent Volume represent the S3 bucket.
+
+```yaml
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: new-pv
+spec:
+  accessModes:
+    - ReadWriteOnce
+  capacity:
+    storage: 5Gi
+  csi:
+    driver: ch.ctrox.csi.s3-driver
+    volumeAttributes:
+      accessKeyID: <YOUR_ACCESS_KEY_ID>
+      secretAccessKey: <YOUR_SECRET_ACCES_KEY>
```

Comment:
Normally you should not have credentials in clear text here, but instead to refer from the PV to a `Secret` that will store those. This is meant to provide separate access controls over the sensitive info.

Reply:
You are right. I'll move s3 credentials into secret, then read it from node server.